frontend/webapp/views/main/templates/bgpupdates.htm

```diff
@@ -382,6 +384,20 @@ <h1>Additional actions</h1>
                                 if(match != null){
                                     filter_parameters[search_table_map[index]] = match[1];
                                 }
+                            }else if(search_table_map[index] == 'as_path'){
+                                var as_path_reg = /^\s*[(\d+)\s(\d+)]*$/;
```

Comment:
I think there is sth wrong with this regex. Why does it demand two integers in? And why are both integers matched as groups? Re-check whether this works properly for any search input on a regex parser (needs to be double-checked)

Reply:
I will update it with ^\s*[(\d+)\s]*$